src/tactic/suggest.lean

```diff
@@ -309,13 +317,21 @@ For performance reasons `suggest` uses monadic lazy lists (`mllist`). This means
 `suggest` might miss some results if `num` is not large enough. However, because
 `suggest` uses monadic lazy lists, smaller values of `num` run faster than larger values.
 -/
-meta def suggest (n : parse (with_desc "n" small_nat)?) : tactic unit :=
-do L ← tactic.suggest_scripts (n.get_or_else 50),
-  when (¬ is_trace_enabled_for `silence_suggest)
-    (if L.length = 0 then
+meta def suggest (n : parse (with_desc "n" small_nat)?)
+  (hs : parse simp_arg_list) (attr_names : parse with_ident_list) (opt : by_elim_opt := { }) :
+  tactic unit :=
+do asms ← mk_assumption_set ff hs attr_names,
+   L ← tactic.suggest_scripts (n.get_or_else 50)
+     {backtrack_all_goals := tt,
+      assumptions := return asms,
```

Comment:
I don't think it's called `assumptions` any more, so you'll have to update this.

Reply:
Oh... `solve_by_elim` has changed as well. Give me a few days to figure out what's happening and make these adjustments and tests.

Reply:
(As I guess you've seen, it's a good idea to `git merge origin/master` before opening a PR.)

Reply:
`git merge origin/master` says "Already up to date."

I've changed `assumptions := return asms,` to `lemma_thunks := return asms,`.